your_daily_dose_malware/backends/malshare.py

```diff
@@ -0,0 +1,46 @@
+import os
+
+import requests
+import click
+from dotenv import load_dotenv
+from rich.progress import Progress
+
+load_dotenv()
+
+
+def scrap_malshare():
+    with Progress() as progress:
+        api_key = os.getenv("MALSHARE_API_KEY")
```

Comment:
Perhaps its an idea on both, to confirm if the API Keys are not empty. For backends that require the API key, to print (Skippping Malshare... no api key) and skipping the rest of the method.

Reply:
done

Reply:
added